src/org/ods/Context.groovy

```diff
@@ -148,9 +148,12 @@ interface Context {
     // ODS Jenkins shared library version, taken from reference in Jenkinsfile.
     String getOdsSharedLibVersion()
 
-    // BitBucket host - value taken from BITBUCKET_HOST.
+    // BitBucket host - value taken from BITBUCKET_URL.
```

Comment:
I guess the old comment `// BitBucket host - value taken from BITBUCKET_HOST` was c

Reply:
Actually it is correct, but I can make the wording clearer.

Reply:
1df927d